apps/meteor/app/file-upload/server/config/AmazonS3.ts

```diff
@@ -64,22 +61,22 @@ const AmazonS3UserDataFiles = new FileUploadClass({
 });
 
 const configure = _.debounce(function () {
-	const Bucket = settings.get('FileUpload_S3_Bucket');
-	const Acl = settings.get('FileUpload_S3_Acl');
-	const AWSAccessKeyId = settings.get('FileUpload_S3_AWSAccessKeyId');
-	const AWSSecretAccessKey = settings.get('FileUpload_S3_AWSSecretAccessKey');
-	const URLExpiryTimeSpan = settings.get('FileUpload_S3_URLExpiryTimeSpan');
-	const Region = settings.get('FileUpload_S3_Region');
-	const SignatureVersion = settings.get('FileUpload_S3_SignatureVersion');
-	const ForcePathStyle = settings.get('FileUpload_S3_ForcePathStyle');
+	const Bucket = settings.get('FileUpload_S3_Bucket') as string;
```

Comment:
```suggestion
	const Bucket = settings.get<string>('FileUpload_S3_Bucket');
```

Reply:
@KevLehman can you please make those improvements for me? I'll need to be AFK for a while now

Reply:
Yup.

Reply:
@KevLehman thanks man